Jenkinsfile

```diff
@@ -94,7 +95,7 @@ builders = pipeline_builder.createBuilders { container ->
                 // Ignore file that crashes cppcheck
                 container.sh """
                                 cd ${project}
-                                cppcheck --enable=all --inconclusive --template="{file},{line},{severity},{id},{message}" ./ -isrc/adc_readout/test/SampleProcessingTest.cpp 2> ${test_output}
+                                cppcheck --enable=all --inconclusive --template="{file},{line},{severity},{id},{message}" ./ -isrc/modules//adc_readout/test/SampleProcessingTest.cpp 2> ${test_output}
```

Comment:
typo.

ALSO, we should delete the lines with AdcSimulator and AdcFileStreamer in this file. Github won't allow me to highlight them though. Line 84+226.

Reply:
well, I got feedback from cppcheck on AdcFileStreamer et al, in my PR, so I very much don't think cppcheck crashes...